<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Departments</class>
 <widget class="QWidget" name="Departments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select a department</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Department Details</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Department Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="txtDepartmentName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Department Head:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txtDepartmentHead">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="cmdSetDepartmenytHead">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="rc.qrc">
            <normaloff>:/images/Users.png</normaloff>:/images/Users.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="cmdAddNew">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/images/Icon_193.ico</normaloff>:/images/Icon_193.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cmdSave">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/images/Icon_10.ico</normaloff>:/images/Icon_10.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cmdDelete">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/images/Icon_136.ico</normaloff>:/images/Icon_136.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
